### ISSUE TITLE
Chrome a11y dev tools fail with auditRulesToRun configuration

### ISSUE THREAD

**Issue by jamsea** [angular/protractor-accessibility-plugin#27]
I originally thought this was related to https://github.com/angular/protractor-accessibility-plugin/pull/26, but I found the same issue here https://github.com/angular/protractor-accessibility-plugin/issues/12#issuecomment-268471486 . I think I've narrowed down the root cause but I'm not sure what the fix is. This bug only happens on the `master` branch with the latest version of protractor.

If I run the plugin with this configuration:
```
            auditConfiguration: {
                auditRulesToRun: [
                    "audioWithoutControls",
                    "badAriaAttributeValue",
                    "badAriaRole",
                    "imagesWithoutAltText",
                    "linkWithUnclearPurpose",
                    "mainRoleOnInappropriateElement",
                    "nonExistentAriaLabelledbyElement",
                    "pageWithoutTitle",
                    "requiredAriaAttributeMissing",
                    "unfocusableElementsWithOnClick",
                    "videoWithoutCaptions"
                ],
                auditRulesToSkip: [
                    "controlsWithoutLabel",
                    "lowContrastElements",
                    "elementsWithMeaningfulBackgroundImage",
                    "focusableElementNotVisibleAndNotAriaHidden"
                ],
                showUnsupportedRulesWarning: false,
                // Ignore controls without label on react-select boxes
                // Ignore <video>
                rules_: {
                    controlsWithoutLabel: {
                        "ignore": [".Select-control", ".form-control", "video"]
                    }
                }
            }
```

I get the following error:

```
[16:01:49] I/plugins -  	 Fail:  protractor-accessibility-plugin Plugin Tests
[16:01:49] E/plugins - 		Failure during teardown: unknown error: Cannot read property 'collectIdRefs' of null
  (Session info: chrome=56.0.2924.87)
  (Driver info: chromedriver=2.27.440174 (e97a722caafc2d3a8b807ee115bfb307f7d2cfd9),platform=Mac OS X 10.11.6 x86_64)
[16:01:49] E/plugins - 		WebDriverError: unknown error: Cannot read property 'collectIdRefs' of null
		  (Session info: chrome=56.0.2924.87)
		  (Driver info: chromedriver=2.27.440174 (e97a722caafc2d3a8b807ee115bfb307f7d2cfd9),platform=Mac OS X 10.11.6 x86_64)
		    at WebDriverError (/Users/james.hush/git/web-webpack/node_modules/selenium-webdriver/lib/error.js:27:5)
		    at Object.checkLegacyResponse (/Users/james.hush/git/web-webpack/node_modules/selenium-webdriver/lib/error.js:505:15)
		    at parseHttpResponse (/Users/james.hush/git/web-webpack/node_modules/selenium-webdriver/lib/http.js:509:13)
		    at doSend.then.response (/Users/james.hush/git/web-webpack/node_modules/selenium-webdriver/lib/http.js:440:13)
		    at process._tickCallback (internal/process/next_tick.js:103:7)
		From: Task: a11y developer tool rules
		    at Driver.schedule (/Users/james.hush/git/web-webpack/node_modules/selenium-webdriver/lib/webdriver.js:816:17)
		    at ProtractorBrowser.executeScriptWithDescription (/Users/james.hush/git/web-webpack/node_modules/protractor/lib/browser.ts:583:24)
		    at runChromeDevTools (/Users/james.hush/git/web-webpack/node_modules/protractor-accessibility-plugin/index.js:172:18)
		    at Object.teardown (/Users/james.hush/git/web-webpack/node_modules/protractor-accessibility-plugin/index.js:66:17)
		    at resolver (/Users/james.hush/git/web-webpack/node_modules/protractor/lib/plugins.ts:460:52)
		    at Function.promise (/Users/james.hush/git/web-webpack/node_modules/q/q.js:682:9)
		    at Plugins.safeCallPluginFun (/Users/james.hush/git/web-webpack/node_modules/protractor/lib/plugins.ts:471:16)
		    at promises.pluginObjs.filter.map.pluginObj (/Users/james.hush/git/web-webpack/node_modules/protractor/lib/plugins.ts:499:28)
		    at Array.map (native)
		    at Plugins.teardown (/Users/james.hush/git/web-webpack/node_modules/protractor/lib/plugins.ts:497:16)
[16:01:49] I/launcher - 0 instance(s) of WebDriver still running
[16:01:49] I/launcher - chrome #01 failed 1 test(s)
[16:01:49] I/launcher - overall: 1 failed spec(s)
[16:01:49] E/launcher - Process exited with error code 1
```

However, if I remove the `auditRulesToRun` rule:
```
            auditConfiguration: {
                auditRulesToSkip: [
                    "controlsWithoutLabel",
                    "lowContrastElements",
                    "elementsWithMeaningfulBackgroundImage",
                    "focusableElementNotVisibleAndNotAriaHidden"
                ],
                showUnsupportedRulesWarning: false,
                // Ignore controls without label on react-select boxes
                // Ignore <video>
                rules_: {
                    controlsWithoutLabel: {
                        "ignore": [".Select-control", ".form-control", "video"]
                    }
                }
            }
```

Everything works fine. I'm guessing it might have something to do with this change https://github.com/angular/protractor-accessibility-plugin/pull/13/files#diff-168726dbe96b3ce427e7fedce31bb0bcR167. My guess is protractor might execute scripts a bit differently now.

Using the same configuration in jsfiddle works completely fine https://jsfiddle.net/jamsea/ejv52v5r/1/ it's only in the plugin where I'm having the issue. For now the "fix" (which might help @FJLopezGarcia) is to remove `auditRulesToRun` and only use `auditRulesToSkip`. If `master` gets published to npm as is there should be a major version bump because of the breaking changes.

**Comment by jdespatis** [comment]
Any news on this problem ?
I encounter the same one with the following versions from my package.json

```
    "protractor": "~5.1.0",
    "protractor-accessibility-plugin": "^0.3.0",
```

**Comment by wyvern8** [comment]
Same on 5.1.2
